ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -87,6 +87,14 @@ function RJSFForm(props) {
     ObjectFieldTemplate = MesheryCustomObjFieldTemplate,
   } = props;
 
+  useEffect(() => {
+    const extensionTooltipPortal = document.getElementById("meshmap-tooltip-portal");
```

Comment:
@Abhishek-kumar09 please use a generic name for to reference this extension point / extension.